packages/@aws-cdk/aws-cloudfront/lib/origin.ts

```diff
@@ -82,7 +79,7 @@ export interface OriginBindOptions {
  *
  * @experimental
  */
-export abstract class Origin implements IOrigin {
+export abstract class BaseOrigin implements IOrigin {
```

Comment:
The convention for abstract base classes is `OriginBase`

Reply:
Yes, of course it is. D'oh. Updated.